x/claim/README.md

```diff
@@ -54,6 +54,21 @@ type ClaimRecord struct {
 
 ```
 
+## A Note on Address Mappings
+
+When an airdrop is created, we call `LoadAllocationData` to load the airdrop data from the allocations file. 
+This will apply `utils.ConvertAddressToStrideAddress` on each of those addresses, and then store those with the `ClaimRecords`.
+For an airdrop to, say, the Cosmos Hub, this will be the proper Stride address associated with that account. 
+`claim` state will only ever store this Stride address.
+
+However, for zones with a different coin type, _this will be an incorrect Stride address_. This should not cause any issues though,
+as this Stride address will be unusable.
+
+In order to claim that airdrop, the user will have to verify that they own the corresponding Evmos address. When the user tries to verify,
```

Comment:
Looking back on this - it might make more sense to key our addresses by `[]byte(addressString)` instead of `AccAddressFromBech32(addressString)` (they both return bytes) 

Definitely not necessary for this upgrade as it'd require a migration. But it'd be a nice future simplification to avoid the scenario of having some correct / some incorrect addresses.

Reply:
Agreed!